lib/manager.rb

```diff
@@ -0,0 +1,22 @@
+
+
+require_relative "reservation"
+require_relative "room"
+
+class Manager
```

Comment:
This class seems pretty small for a "Manager" class which coordinates the reservations etc.  I would also expect that it would have a list of reservations.